### PR TITLE
Solution #1

### PR DESCRIPTION
**Pull request by erikagayan**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,10 +1,46 @@
...
+        else:
+            guild = None
+
+        Player.objects.create(
+            nickname=player_name,
+            email=player_info["email"],
+            bio=player_info["bio"],
+            race=race,
+            guild=guild,
+        )
```
Comment (RedCatHot) [inline main.py:43]: Everything is okay, but if u wanna, u can try to split this into functions
Maybe mentor will ask u for that

**Review by RedCatHot — CHANGES_REQUESTED** [review]
U forgot to add migrations, cause of that tests failed. U can do this even here, just add file into git project, and I'll aprove this

**Review by RedCatHot — APPROVED** [review]

File: main.py [hunk 2]
```diff
@@ -1,10 +1,46 @@
...
+        race = player_info["race"]
+        guild = player_info["guild"]
+
+        race, is_race_created = Race.objects.get_or_create(
+            name=race["name"],
+            description=race["description"]
+        )
+
+        if is_race_created:
+            for skill in player_info["race"]["skills"]:
```
Comment (LiudmylaKulinchenko) [inline main.py:22]: You will get an error if skill is already exists, you should check it

Comment (LiudmylaKulinchenko) [reply main.py:22]: fix in guild, race and user

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

File: main.py (hunk 2 above)
Comment (LiudmylaKulinchenko) [reply main.py:22]: Don't forget to use `race` variable here

**Review by Oleg-s-s — CHANGES_REQUESTED** [review]
It would be better to split main.py into functions to make you code more modular

File: main.py [hunk 3]
```diff
@@ -1,10 +1,58 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
+def create_races(players: dict) -> None:
```
Comment (LiudmylaKulinchenko) [inline main.py:8]: consider to change function names. You not just create races, but also skills. Same in the next func

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]
Well done

**Review by YuriiKindrat — APPROVED** [review]

**Pull request closed without merging**